chains/ngmi/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://ngmi-2712765173321000-1.sagaexplorer.io/api
+    family: blockscout
+    name: NGMI Explorer
+    url: https://ngmi-2712765173321000-1.sagaexplorer.io
+blocks:
+  confirmations: 1
+  estimateBlockTime: 5
+  reorgPeriod: 0
+chainId: 2712765173321000
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: NGMI
+domainId: 2712765173321000
+gasCurrencyCoinGeckoId: ngmi
+name: ngmi
+nativeToken:
+  decimals: 18
+  name: NGMI
+  symbol: NGMI
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc.saga.xyz/chainlets/ngmi_2712765173321000-1
```

Comment:
this rpc doesn't work, can't find alternatives either

is there a docs page for this chain?

Reply:
found https://app.saga.xyz/chainlets/ngmi_2712765173321000-1